test/Altinn.Platform.Events.Tests/appsettings.json

```diff
@@ -3,7 +3,7 @@
     "EnableDBConnection": "false"
   },
   "GeneralSettings": {
-    "Hostname": "localhost:5080",
+    "BaseUri": "https://platform.localhost:5080",
```

Comment:
```suggestion
    "BaseUri": "http://localhost:5080",
```

Reply:
Fixed.

Reply:
I tried replying to these comments using the new (built-in?) support in VS Code. Didn't seem to work quite right. 

I've updated all the tests to use this Uri